controller/search_blackbox_test.go

```diff
@@ -575,6 +577,19 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 
 	wirepo := workitem.NewWorkItemRepository(s.DB)
 
+	labelNames := []string{"IMPORTANT", "backend", "UI", "REST"}
+	testFixtures, err := tf.NewFixture(s.DB, tf.Labels(4, func(f *tf.TestFixture, idx int) error {
+		f.Labels[idx].Name = labelNames[idx]
```

Comment:
@pranavgore09 Great to see this usage of the test fixture. Thank you for proving that it has its purpose!!! But you can use the `tf.NewTestFixture` and pass in the test to automatically fail the test if there was an error. It is just a convenience function.

Reply:
Thanks @kwk, you wrote it I just used it.
Using `TestFixtures` when possible... https://github.com/fabric8-services/fabric8-wit/pull/1640/commits/46a3fe097d1ef83dfab077182e2ed21aee75c515

One thing is, should we add "collaborators" to the fixture?

Reply:
> One thing is, should we add "collaborators" to the fixture?

I think @xcoulon mentioned that @alexeykazakov wants to refactor this. @alexeykazakov does this hold true?

Reply:
Well, we don't store collaborators in WIT DB. Not sure what you are referring to when asking about adding collaborators to the test fixture. What we do store in WIT is a reference to Keycloak resources which should be used to check if a user is a collaborator in some space. And we are going to remove these references from WIT later this week or next week. We are also removing the collaborator management API (add, update, remove) from WIT.
The only thing which will remain in WIT (for awhile) is auth.Authorize(ctx, spaceID) function which will call Keycloak to check user's permissions in the space.
So, don't bother to add anything related to collaborators to the test fixture until it's really necessary for you test.

Reply:
@alexeykazakov thanks for the information.